lib/index.js

```diff
@@ -55,6 +56,55 @@ exports.plugin = {
             server.ext('onPreStart', internals.initialize);
             server.ext('onPostStop', internals.stop);
 
+            // Expose an hpal command for schwifty-migration
+            server.expose('commands', {
+                'migrate:diff': (initializedServer, args, rootDir, ctx) => {
+
+                    const SchwiftyMigrateDiff = require('./schwifty-migrate-diff').get();
+
+                    if (!SchwiftyMigrateDiff) {
+                        throw new ctx.DisplayError('To use this command please first install `npm install --save-dev schwifty-migrate-diff`');
+                    }
+
+                    const [mode, migrationName, argsMigrationsDir] = args;
+
+                    const realmsWithMigrationsDir = internals.state(initializedServer.realm).collector.realmsWithMigrationsDir;
+
+                    const absRootDir = Path.isAbsolute(rootDir) ?
+                        Path.normalize(rootDir) :
+                        Path.join(process.cwd(), rootDir);
+
+                    const realmsWithMigrationsInRoot = Array.from(realmsWithMigrationsDir).reduce((collector, realm) => {
+
+                        const migrationsDir = internals.state(realm).migrationsDir;
+                        if (migrationsDir.includes(absRootDir)) {
```

Comment:
I would suggest at least using `startsWith()`.  But you can also use this approach from hpal,
```js
            const partIsDotDot = (part) => part === '..';
            const isChildPath = (pathA, pathB) => { // A child of B

                const relParts = Path.relative(pathA, pathB).split(Path.sep);

                return relParts.every(partIsDotDot);

            };
```

Reply:
didn't even know `startsWith` and `endsWith` was a thing!

Reply:
Pretty slick stuff!  It doesn't work well four our purposes in some cases.  For example `/my/direction-is-weird` isn't a child path of `/my/directory` even though it `startsWith()` the same string.